packages/gestalt/src/SearchField.css

```diff
@@ -47,22 +46,13 @@ html[dir="rtl"] .inputActive {
 
 .clear {
```

Comment:
This needs visual snapshot updates no?

Reply:
The visual snapshot doesnt cover the "with content" state just the empty state. The button only shows up when there's input

Reply:
I asked designers if they were ok with using IconButton here and shared visual changes. @cintiapub and @hectoid were ok with it as with VR and more squared roundings the icon sizing wont be as much rellevant